ros_ws/src/autonomous/reinforcement_learning/scripts/drive_policy_gradient.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
+
+from reinforcement_learning_node import ReinforcementLearningNode
+import os
+import rospy
+from parameters_policy_gradient import Policy, ACTIONS, LASER_SAMPLE_COUNT
+import torch
+from torch.distributions import Categorical
+
+
+class QLearningDrivingNode(ReinforcementLearningNode):
```

Comment:
Why is this called "QLearningDrivingNode" when its purpose is to drive with policygradiant?